kotlinpoet/src/test/java/com/squareup/kotlinpoet/LambdaTypeNameTest.kt

```diff
@@ -192,4 +192,40 @@ class LambdaTypeNameTest {
     assertThat(typeName.toString())
       .isEqualTo("((kotlin.Int) -> kotlin.Int) -> kotlin.Unit")
   }
+
+  @Test fun equalsAndHashCode() {
+    val lambdaTypeName1 = LambdaTypeName.get(
+      parameters = arrayOf(Int::class.asTypeName()),
```

Comment:
Can use `INT` and `ANY`, declared inside `TypeName`.

Reply:
Thanks, done as 53c75f56